.github/workflows/3x.yml

```diff
@@ -128,7 +128,7 @@ jobs:
       - name: Full Composer Install
         run: composer install
       - name: Prepare needed folder and fix permissions
-        run: mkdir -p ~/.terminus/cache/tokens && chmod +x ./terminus
+        run: mkdir -p ~/.terminus/cache/tokens && mv ./terminus.phar ./terminus && chmod +x ./terminus
```

Comment:
Should `chmod +x ./terminus` be here?  Should it be in the permissions section near [here](https://github.com/pantheon-systems/terminus/pull/2273/files#diff-c0a3c210f3e9217a0d2bbccd44436d0595423996f523bedf01e93574263c7e5aR25) instead?

Reply:
They're separate things.  This is part of the linux packaging job.  That's the creation of the terminus binary that happens before testing.

Reply:
All good.  Was more of a curiosity than anything.